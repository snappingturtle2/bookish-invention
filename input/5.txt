1
HEX_STRING
111111111111111
DUMP
MUL_APINT
0 0 0
DUMP
END
