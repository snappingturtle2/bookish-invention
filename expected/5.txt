0x0111111111111111

0x0123456789abcdefedcba987654321

